GameOfRunes/GameOfRunes/Custom Nodes/BackgroundNode.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  BackgroundNode.swift
+//  GameOfRunes
+//
+//  Created by Andy on 13/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+import Foundation
+import SpriteKit
+
+class BackgroundNode: SKSpriteNode {
```

Comment:
Would it be better to decouple the gesture recognising functionality from the background? Perhaps a SKnode that sits on top of the enemy area? Or an ECS component?

Reply:
Refactored to `GestureAreaNode`